instagram/src/components/SearchBar/SearchBar.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import "./SearchBar.css";
+
+const SearchBar = props => {
+  return (
+    <div className="search-bar-wrapper">
+      <div className="social">
+        <i className="fab fa-instagram fa-5x" />
+        <h2 className="instagram-title">Instagram</h2>
+      </div>
+
+      <div>
+        <input type="text" placeholder="Search" onKeyDown={props.searchPosts} />
```

Comment:
are you sure you want to use the onKeyDown event? How do we usually handle input changes?